libs/order/src/models/order.ts

```diff
@@ -0,0 +1,23 @@
+import { DaffOrderItem } from './order-item';
+import { DaffOrderAddress } from './order-address';
+import { DaffOrderPayment } from './order-payment';
+import { DaffOrderCoupon } from './order-coupon';
+import { DaffOrderTotal } from './order-total';
+import { DaffOrderShipment } from './order-shipment';
+import { DaffOrderInvoice } from './order-invoice';
+
+export interface DaffOrder {
+  id: number | string;
+  customer_id: number | string;
+  created_at: string;
+  updated_at: string;
+  status: string;
+  totals: DaffOrderTotal[];
+  applied_codes: DaffOrderCoupon[];
+  items?: DaffOrderItem[];
+  addresses?: DaffOrderAddress[];
+  shipments?: DaffOrderShipment[];
+  payment?: DaffOrderPayment;
+  invoices?: DaffOrderInvoice[];
+  credits?: DaffOrderInvoice[];
```

Comment:
I think `credits` should be a rebind of a `DaffOrderInvoice` to a `DaffOrderCredit`

Reply:
This too would be an alias.

Reply:
changed